src/xml_processing/tests/inputs/alliance_pmids

```diff
@@ -1,6 +1,6 @@
 2
-10206683
```

Comment:
This file also shouldn't change, but this seems like it hasn't really changed, just moved an indentifier further down